ui/test/utils/wrappers.test.ts

```diff
@@ -1,17 +1,17 @@
-import {get} from 'src/utils/wrappers'
+import {getNested} from 'src/utils/wrappers'
```

Comment:
Really just semantics but what about `getDeep`?

Reply:
sounds good to me